lib/runner/filematcher.js

```diff
@@ -53,15 +53,20 @@ module.exports = {
       var match = true;
 
       if (!Array.isArray(moduleTags)) {
-        return typeof opts.tag_filter == 'undefined' && typeof opts.skiptags != 'undefined';
+        return typeof opts.tag_filter == 'undefined' && typeof opts.tags_filter == 'undefined' && typeof opts.skiptags != 'undefined';
       }
 
-      if (opts.tag_filter || opts.skiptags){
+      if (opts.tag_filter || opts.tags_filter || opts.skiptags){
         moduleTags = this.convertTagsToString(moduleTags);
 
         if (opts.tag_filter) {
           match = this.containsTag(moduleTags, this.convertTagsToString(opts.tag_filter));
         }
+        
+        if (opts.tags_filter) {
```

Comment:
have not tested it, how would you expect it to work? 
I would expect, that a valid test has one of the `--tag` filters OR the `--tags` filter, so they could be used together. But others might assume an AND behavior. 
Maybe we should throw at least a warning if both are used together.